packages/markdown/components/Code/index.jsx

```diff
@@ -5,12 +5,32 @@ const PropTypes = require('prop-types');
 // apps. Necessary because of people like this:
 // https://github.com/codemirror/CodeMirror/issues/3701#issuecomment-164904534
 const syntaxHighlighter = typeof window !== 'undefined' ? require('@readme/syntax-highlighter') : false;
+const copy = require('copy-to-clipboard');
+
+const remapLang = {
+  c: 'cplusplus',
+  'c++': 'cplusplus',
+  cpp: 'cplusplus',
+  docker: 'dockerfile',
+  html: 'htmlmixed',
+  js: 'javascript',
+  py: 'python',
+  sh: 'shell',
+  bash: 'shell',
+  mysql: 'sql',
+};
 
 function Code(props) {
-  const { className, children, lang, meta } = props;
-  const language = (className || '').replace('language-', '');
```

Comment:
Using the pre-parsed `meta.lang` attribute, instead of splitting the `className` attribute to figure it out on the fly...